src/cascadia/WindowsTerminal/pch.h

```diff
@@ -62,6 +62,7 @@ Module Name:
 //  * ApplicationModel for finding the path to wt.exe
 #include <winrt/Windows.UI.Core.h>
 #include <winrt/Windows.UI.Xaml.Controls.h>
+#include <winrt/Windows.UI.Xaml.Controls.Primitives.h>

```

Comment:
Curious: we need this for `XamlRoot`?

Reply:
Nah, we need it for [`Popup`](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.popup?view=winrt-20348)